awscli/examples/rum/get-app-monitor-data.rst

```diff
@@ -0,0 +1,19 @@
+**To retrieve the raw performance events that RUM has collected from your web application**
+
+The following ``get-app-monitor-data`` retrieves the raw performance events that RUM has collected from your web application. ::
+
+	aws rum get-app-monitor-data \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general